corehq/apps/app_manager/models.py

```diff
@@ -4668,6 +4668,9 @@ def make_reversion_to_copy(self, copy):
 
         return app
 
+    def profile_url(self):
```

Comment:
is this supposed to be called like `self.profile_url()` or `self.profile_url`? Seems like currently it would have to be called like the former?